### PR TITLE
Process messages in a task

### PR DESCRIPTION
**Pull request by ashkan18**
# Problem
With current approach when there is an issue in processing the incoming event, since this code is in GenServer it causes the GenServer to go down and it will keep retrying, but since that event with issue wasn't processed it will try to process it again which eventually ends up in stopping connection on that topic

# Solution
Process the event in a `Task` so failures wouldn't impact GenServer and also in general it's a good practice to don't hold GenServer while we are processing the event.

### CODE REVIEW

File: lib/services/slack_command_service.ex [hunk 1]
```diff
@@ -29,7 +29,7 @@ defmodule Aprb.Service.SlackCommandService do
         "Subscribed topics: #{current_topics}"
 
       Regex.match?( ~r/unsubscribe/ , params[:text])  ->
-        [command | topic_names] = String.split(params[:text], ~r{\s}, parts: 2)
+        [_command | topic_names] = String.split(params[:text], ~r{\s}, parts: 2)
```
Comment (izakp) [inline lib/services/slack_command_service.ex]: Why the change to `_command`?

File: lib/services/amq_event_service.ex [hunk 2]
```diff
@@ -64,7 +64,7 @@ defmodule Aprb.Service.AmqEventService do
   defp consume(channel, topic, tag, redelivered, payload) do
     try do
       Basic.ack channel, tag
-      if acceptable_message?(payload), do: EventService.receive_event(payload, topic)
+      if acceptable_message?(payload), do: Task.async(fn -> EventService.receive_event(payload, topic) end)
```
Comment (izakp) [inline lib/services/amq_event_service.ex]: Nice!

File: lib/services/slack_command_service.ex (hunk 1 above)
Comment (ashkan18) [inline lib/services/slack_command_service.ex]: just a cleanup thing, we don't use `command` so Elixir suggests to add `_`

**Comment by ashkan18** [comment]
@izakp going to merge this so we can try this and https://github.com/artsy/aprb/pull/76 on staging
